.github/workflows/ios_deploy_staging_to_firebase.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
       FIREBASE_CLI_TOKEN: ${{ secrets.FIREBASE_CLI_TOKEN }}
       FIREBASE_APP_ID_STAGING: ${{ secrets.FIREBASE_IOS_APP_ID_STAGING }}
+      GITHUB_RUN_NUMBER: ${{ secrets.GITHUB_RUN_NUMBER }}
```

Comment:
could you please move this down or up to have better grouping for Firebase stuff?